docs/sorting-algorithms/selection-sort.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Selection Sort
+slug: /sorting-algorithms/selection-sort
+---
+
+## Introduction
+
+Selection sort is a sorting algorithm that finds out the smallest element in the unsorted part in each iteration and puts it to the last of the sorted part. It does less memory writes as compared to other sorting algorithms. It is not stable and In place sorting algorithm. It takes the same time in every case `(best, average, worst)`.
+
+## Explanation 
+
+First, we will find out the smallest element in the range `[0,n-1]` and swap the smallest element with the element at index `0`. Then we will find the smallest element in the range `[1,n-1]` and swap it will the element at index `1` and we keep on reducing the range and swapping the smallest element with the first element in the range till the array gets sorted.
+
+Let's have one example: 
+```
```

Comment:
Add a line stating: `Let's have an array arr[] where we have the following elements: 5 4 3 2 1`

Reply:
done